examples/gRPC/util.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+import grpc
+
+
+def get_gRPC_channel(target="prod"):
```

Comment:
More inconvenient for you, but easier for most other users :grin: 
```suggestion
def get_gRPC_channel(target="local"):
```